_episodes/04-recipe.md

```diff
@@ -273,9 +273,20 @@ Please note the following sections:
 
 > ## Running ESMValTool
 >
-> Use the command:
+> The example recipe presented above requires a single data file. If you are not
+> working on a system with data available, you can download the file with the
+> command:
+>
+> ~~~bash
+> wget http://esgf-data1.ceda.ac.uk/thredds/fileServer/esg_dataroot/cmip5/output1/MOHC/HadGEM2-ES/historical/mon/ocean/Omon/r1i1p1/v20110916/thetaoga/thetaoga_Omon_HadGEM2-ES_historical_r1i1p1_185912-200512.nc
+> ~~~
```

Comment:
This section might be better placed on setup.md 

Also if we tell user to download to `~/default_inputpath` we can reduce some typing.

Reply:
I fear that people will not be able to find it in that very long document.

Reply:
see #103

Reply:
Once #104 is merged we should point to that section of the setup.md instead of having the command here.

Reply:
Let's wait for that before continuing with this PR then